app/serializers/organization_serializer.rb

```diff
@@ -1,10 +1,12 @@
+require "tasks_visitors/inject_strings"
```

Comment:
Do we really need this require? Rails constant autoloading should find this..

Reply:
All the org and project serializer specs pass without it, so I guess not.